denote.el

```diff
@@ -1744,28 +1742,32 @@ Subroutine of `denote--file-with-temp-buffer'."
 (defun denote-retrieve-front-matter-title-value (file file-type)
   "Return title value from FILE front matter per FILE-TYPE."
   (denote--file-with-temp-buffer file
-    (when (re-search-forward (denote--title-key-regexp file-type) nil t 1)
+    (when (and file-type
```

Comment:
Is it not better to have the check for the `file-type` before the `denote--file-with-temp-buffer`?

Reply:
You are right! I will fix this in my next pull request.